src/main/java/interface_adapters/Sell/SellState.java

```diff
@@ -57,6 +72,30 @@ public String getSellSuccess() {
         return sellSuccess;
     }
 
+    public void setBalance(Double balance) {
```

Comment:
- Your whole PR is an excellent example of good naming conventions, with attributes and methods being named using `camelCase` and classes named using `PascalCase`. 
- The changes you made to `SellState` were very crucial, as without them the view would have been much less functional.
- The code's correctness is undeniable, as it is so neat that I could understand every addition it will make, but also it passes all of our Maven tests which would detect compile errors and failures of other pre-built tests we have made. 
- The test coverage of the `Sell` use case and interface adapters is not extensive, however this is not a problem specific to this use case but a general problem for our app. I just wanted to mention that we will need significantly more testing, including on the files that are modified here. 
- As with every class in our app, these changes reflect very good SOLID design principles and CA setup. 
- Another thing to note with the `Sell` use case here is that Javadoc is missing. Every constructor is public and many of the methods, including this one, `SellState.setBalance` are also public, meaning all of these require Javadoc.
- Your explanation/description of this PR was exemplary, as it laid out the changes you made and the reasons for making them. 

Overall, this is a high-quality PR, and thus merits approval.